### PR TITLE
[DO-1229] Full word filter

### PR DESCRIPTION
**Pull request by Iva-khar**
Before:
2df77b0d-4ea8-4281-8b09-6a2eeadcc9cb
After:
Screenshot from 2021-05-13 14-15-24

### CODE REVIEW

**Review by roman-tiukh — COMMENTED** [review]
image

**Comment by Iva-khar** [comment]
Writed self FullWordSearchFilter

File: business_register/filters.py [hunk 1]
```diff
@@ -279,3 +280,8 @@ class Meta:
 
 class HistoricalCompanyRelatedFilterSet(filters.FilterSet):
     history_date = filters.DateFromToRangeFilter()
+
+
+class FullWordSearchFilter(SearchFilter):
+    def get_search_terms(self, request):
+        return [request.query_params.get(self.search_param, '')]
```
Comment (roman-tiukh) [inline business_register/filters.py:287]: where? 
```python
params = params.replace('\x00', '')
```
its useful

**Review by roman-tiukh — CHANGES_REQUESTED** [review]

File: business_register/filters.py (hunk 1 above)
Comment (Iva-khar) [reply business_register/filters.py]: Added

**Review by roman-tiukh — APPROVED** [review]

**Pull request merged**